src/index.html

```diff
@@ -19,9 +19,381 @@
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__nav-item"
+            data-qa="nav-hover"
+          >
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Laptops&nbsp;&&nbsp;Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="catalog__card-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+        />
+
+        <h2 class="catalog__card-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="catalog__card-code">Product code: 195434</p>
+
+        <div class="catalog__card-rating">
+          <div class="catalog__card-stars">
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+            <div class="catalog__card-stars-star"></div>
+          </div>
```

Comment:
You star block from the previous task
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```